src/components/layout/mainLayout.tsx

```diff
@@ -39,43 +42,35 @@ const MainLayout: React.FC<Readonly<Props>> = ({
 };
 
 const Navigation = () => {
+  const currentPath = usePathname();
```

Comment:
```suggestion
  const currentPath = usePathname() as AppRoutes;
```

Could this work so you can avoid casting on all other places?